.github/workflows/ci.yaml

```diff
@@ -0,0 +1,62 @@
+name: CI
+on:
+  push:
+    branches:
+      - master
+      - release
+  pull_request:
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ['3.6', '3.7', '3.8']
+    defaults:
+      run:
+        shell: bash -l {0}
+    steps:
+    - uses: actions/checkout@v2
+    - uses: conda-incubator/setup-miniconda@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+        mamba-version: "*"
+        channels: conda-forge,bioconda
+        channel-priority: true
+        activate-environment: test
+    - run: mamba install mafft raxml fasttree iqtree vcftools pip numpy
+    - run: pip install biopython==1.67
```

Comment:
Seeing this pinned BioPython version reminded me of @tsibley's [recent comments about using a test matrix for BioPython (and pandas) versions](https://github.com/nextstrain/augur/issues/798#issuecomment-993044873). It's outside the scope of this PR, but if we wanted to test multiple BioPython versions with GitHub Actions, how would we implement that? Would it just be an entry in the `matrix` dictionary above?

Reply:
`matrix` is for testing all combinations of various entries. We can start with that and see how it works. 
There could be some compatibility issues since for example [BioPython 1.67 only has official support for up to Python 3.5](https://github.com/biopython/biopython/blob/master/NEWS.rst#8-june-2016-biopython-167). It might be more appropriate to use [conditional steps](https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#jobsjob_idif) for different BioPython versions based on Python version.